### PR TITLE
Apply patch for GMP CVE-2021-43618

### PR DESCRIPTION
**Pull request by ViralBShah**
The checksums need to be updated before this is merged.

### CODE REVIEW

File: deps/Versions.make [hunk 1]
```diff
@@ -22,7 +22,7 @@ DSFMT_VER := 2.2.4
 DSFMT_JLL_NAME := dSFMT
 
 # GMP
-GMP_VER := 6.2.1
+GMP_VER := 6.2.1+2
```
Comment (giordano) [inline deps/Versions.make:25]: This shouldn't be touched:
```suggestion
GMP_VER := 6.2.1
```
Instead you have to update the version number in the Project.toml of the stdlib

**Review by giordano — COMMENTED** [review]
See for example #44810 for the steps to update a JLL. I guess we should record this information somewhere, I have always to write down what I do in the pull requests so that I remember

File: deps/Versions.make (hunk 1 above)
Comment (ViralBShah) [reply deps/Versions.make:25]: Thanks - this is what I was missing.

**Comment by ViralBShah** [comment]
Good to merge?

**Pull request merged**